Microsoft.Toolkit.Uwp.Services/Services/Facebook/FacebookService.cs

```diff
@@ -21,6 +21,7 @@ namespace Microsoft.Toolkit.Uwp.Services.Facebook
     /// <summary>
     /// Class for connecting to Facebook.
     /// </summary>
+    [Obsolete("The underlying library these helpers rely on is not currently maintained.")]
```

Comment:
This message is a bit unclear to me. 
What helpers? Just the FB service? all of WCT?  I think it should be scoped if possible to FB.
What underlying library? Is the FB library or libraries in the WCT?  scoped message is better.

Reply:
Also should you add a comment in the sample.js file above as well. There's a Sample for FB Service on line 620 of that file.

Reply:
Thanks Herrick, I'll clarify. I think it's already tagged as deprecated in the sample, but maybe not with an additional message, so I'll double-check that.